ludwig/datasets/mixins/kaggle.py

```diff
@@ -0,0 +1,38 @@
+import os
+from pathlib import Path
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+DEFAULT_CACHE_LOCATION = str(Path.home().joinpath('.ludwig_cache'))
+api = KaggleApi()
```

Comment:
Let's call this within `download_raw_dataset` so we give the user a chance to fix the environment without needing to recreate the interpreter state.

Reply:
Fixed

Reply:
Did you push your changes?  The code here still has the previous behavior.

Reply:
I did actually and looking at this locally shows this function